tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -21,20 +21,25 @@
     <!-- Latest package https://www.nuget.org/packages/Aerospike.Client/6.0.1 -->
     <PackageReference Include="Aerospike.Client" Version="6.0.1" />
 
-    <!-- Integration: System.Web.Mvc -->
-    <!-- Assembly: System.Web.Mvc -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.AspNet.Mvc/5.2.9 -->
-    <PackageReference Include="Microsoft.AspNet.Mvc" Version="5.2.9" />
```

Comment:
Can you restore the entries that reference `Microsoft.AspNet.Mvc` and `System.ServiceModel.Http`? This might have been a result of the rebasing I just commented on in the definitions json file

Reply:
Running ./build.sh GeneratePackageVersions caused this to get deleted see [b32278e](https://github.com/DataDog/dd-trace-dotnet/pull/4084/commits/b32278e5136cbb0a25cf70046dbc18c66cc23976)